<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewer</class>
 <widget class="QDialog" name="ImageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Btn_hf0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>hf0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_hf1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>hf1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_hf2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>hf2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_vf0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>vf0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_vf1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>vf1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_vf2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>vf2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_hc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>hc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_vc">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>vc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_bk">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>bk</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera Image</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera Left/Right</string>
   </property>
   <widget class="QRadioButton" name="radio_camL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera Left</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_camR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera Right</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_browse">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <zorder>groupBox_1</zorder>
  <zorder>Btn_hf0</zorder>
  <zorder>Btn_hf1</zorder>
  <zorder>Btn_hf2</zorder>
  <zorder>Btn_vf0</zorder>
  <zorder>Btn_vf1</zorder>
  <zorder>Btn_vf2</zorder>
  <zorder>Btn_hc</zorder>
  <zorder>Btn_vc</zorder>
  <zorder>Btn_bk</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>lineEdit</zorder>
  <zorder>Btn_browse</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
